benchmark/benchmark.py

```diff
@@ -595,18 +596,19 @@ def run_test_real(
     show_fnames = ",".join(map(str, fnames))
     print("fnames:", show_fnames)
 
-    coder = Coder.create(
-        main_model,
-        edit_format,
-        io,
-        fnames=fnames,
-        use_git=False,
-        stream=False,
-        verbose=verbose,
-        # auto_lint=False,  # disabled for code-in-json experiments
-        cache_prompts=True,
-        suggest_shell_commands=False,
-    )
+    with change_dir(testdir):
```

Comment:
If we don't do this, the CEDARScript editor won't know which directory to consider as the base. We can remove all changes to this file, but then we won't be able to run benchmarks using `cedarscript-g` edit format.

Reply:
I think this is fine, since
- the benchmark does not touch regular usage
- it is more or less part of the new coder

but @paul-gauthier should make the call of course.